x-pack/plugins/cases/server/common/types.ts

```diff
@@ -5,10 +5,17 @@
  * 2.0.
  */
 
+import { KueryNode } from '../../../../../src/plugins/data/server';
+import { SavedObjectFindOptions } from '../../common/api';
```

Comment:
I believe with @stephmilovic's changes, you can just import from `/common` now

Reply:
Oh really? I remember her PR but it got reverted. We need to merge with master again on our feature branch (`cases-rbac-poc`) and fix the conflicts :)